tests/integration/submissions/test_submissions_views.py

```diff
@@ -276,6 +276,8 @@ def test_update_author(app, api_client, create_user, requests_mock):
         == history.headers["Authorization"]
     )
     assert history.url == f"{app.config['INSPIRE_NEXT_URL']}/workflows/authors"
+    assert "datetime" in post_data["data"]["acquisition_source"]
+    del post_data["data"]["acquisition_source"]["datetime"]
```

Comment:
can we just use freezetime?

Reply:
why we removed ``freeze_time``?

Reply:
I didn't removed anything, Rebase merged it in a weird way. Fixed now.